pkg/logger/logger.go

```diff
@@ -32,11 +31,11 @@ func SetLogLevel(logLevel string) (zap.Opts, error) {
 	var level zapcore.Level
 	switch strings.ToUpper(logLevel) {
 	case "DEBUG":
-		level = zapcore.Level(-1)
+		level = zapcore.DebugLevel
 	case "INFO":
-		level = zapcore.Level(0)
+		level = zapcore.InfoLevel
 	case "ERROR":
-		level = zapcore.Level(1)
+		level = zapcore.ErrorLevel
```

Comment:
previously Level(1) was warn, do we want to change this?

Reply:
We could use warnlevel but it doesn't change the test or loggers behavior so I figured use the matching level. Zap is more granular than what the logr interface supports,  afaict.